docs/_user_guide/sas.md

```diff
@@ -0,0 +1,51 @@
+# Running SAS Interactively
+
+On the Yens, SAS is available via the `module` command. 
+
+See all currently installed versions with:
+
+```title="Terminal Input"
+module avail sas
+```
+
+You will see the current SAS versions listed:
+
+```{.yaml .no-copy title="Terminal Output"}
+
+-------------------------------------------- Global Aliases --------------------------------------------
+
+
+---------------------------------------- /software/modules/Core ----------------------------------------
+   sas/9.4
+```
+
+## Load SAS Module
+To use SAS software on the Yens, load the module:
+```title="Terminal Input"
+ml sas
+```
+This will load the default `(D)` module. You can also load a specific version of SAS with:
```

Comment:
we only have one sas version and there is no D in the output so this may be confusion

Reply:
cleared up